src/main/java/fauna/sample/controllers/orders/OrdersController.java

```diff
@@ -160,6 +160,10 @@ Future<Order> update(@PathVariable String id, @RequestBody OrderUpdate req) {
     Future<Page<Order>> getByCustomer(@PathVariable("id") String customerId, @RequestParam(required = false) String afterToken, @RequestParam(required = false) Integer pageSize) {
         Query query;
         if (afterToken != null) {
+            /**
+             * Fauna after token may contain special characters (i.e.+, -,&). 
+             * Make sure to encode your URL properly from the client side when calling this API
+             */
```

Comment:
Looks — I made some minor suggestions. It may make more sense to move this higher — right above 
`@RequestParam(required = false) String afterToken,` since that's where the param is defined. This code is just handling.
```suggestion
            /**
             * The `afterToken` parameter contains a Fauna `after` cursor.
             * `after` cursors may contain special characters, such as `.` or `+`). 
             * Make sure to URL encode the `afterToken` value to preserve these
             * characters in URLs.
```